BlueprintUI/Sources/Environment/EnvironmentKey.swift

```diff
@@ -26,4 +26,27 @@ public protocol EnvironmentKey {
     /// The default value that will be vended by an `Environment` for this key if no other value
     /// has been set.
     static var defaultValue: Self.Value { get }
+
+    /// Equivalency check on the `Value`s of `EnvironmentKey`s
```

Comment:
It would be good to explain what "equivalency" means here; eg "return false if the difference in values could affect the measurement or layout of a consuming element..."